backoffice/src/Services/Enums.cs

```diff
@@ -0,0 +1,13 @@
+namespace Droits.Services;
+
+public enum EmailTemplateType
+{
+    ReportAcknowledged,
+    ReportCompleteInformation,
+    ReportConfirmed,
+    LloydsOfLondonNotification,
+    ClosureOwnerNotFound,
+    ClosureOwnerWaivesRights,
+    ClosureCustom,
+    CustomEmail
```

Comment:
There should be another template type here which we added in commit f6da5d2745958fd48253d25a702774bc96475e2d